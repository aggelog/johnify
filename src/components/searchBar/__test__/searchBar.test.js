import React from 'react';
import SearchBar from '../index';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<SearchBar changeSearchTextHandler={jest.fn()} performSearch={jest.fn()} searchValue="" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});