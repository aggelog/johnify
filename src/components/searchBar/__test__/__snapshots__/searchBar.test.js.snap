// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="search-bar__container fadeIn"
>
  <form
    onSubmit={[Function]}
  >
    <input
      className="search-bar__input-field"
      onChange={[Function]}
      placeholder="This is where the magic begins.."
      type="text"
      value=""
    />
  </form>
</div>
`;
